internal/distro/rhel85/distro_test.go

```diff
@@ -30,20 +30,23 @@ func TestArchitecture_ListImageTypes(t *testing.T) {
 		rhelAdditionalImageTypes []string
 	}{
 		{
-			arch:     "x86_64",
-			imgNames: []string{},
+			arch: "x86_64",
+			imgNames: []string{
+				"tar",
+				"tar-installer",
+			},
```

Comment:
From what I can see this is verifying that we've got these 2 new image types. However I don't see any test which tries to build the new image types and makes sure nothing fails. `cmd/osbuild-image-tests/main_test.go` may be a good place to start. Alternatively a separate integration test would do as well.

Reply:
Thanks for the review, Alex.  I was thinking an integration test that builds the installer and auto-installs it, similar to the ostree integration tests, would be a good idea for the tar-installer.

The plain tar image type existed in the previous distros, so we could follow the same testing here.

Reply:
@achilleas-k please open a ticket in JIRA with the `QE` prefix for this tar integration test if you haven't already so that we don't forget about it and make sure to plan for the work accordingly.

Reply:
^^^ I see installers.sh. Closing this comment.